### PR TITLE
Add an init container which install wordpress, from some credentials

### LINKED ISSUE #11: Bootstrap wordpress on first run

### PR DESCRIPTION
**Pull request by vtemian**
Installs wordpress from `WORDPRESS_BOOTSTRAP_USER ` and `WORDPRESS_BOOTSTRAP_PASSWORD`.

Closes #11 

#### Tests
* create a new site that has `WORDPRESS_BOOTSTRAP_USER ` and `WORDPRESS_BOOTSTRAP_PASSWORD`
* it **should** get installed by the operator
* `WORDPRESS_BOOTSTRAP_USER ` and `WORDPRESS_BOOTSTRAP_PASSWORD`**shouldn't** be present in the site's pods env.

### CODE REVIEW

File: pkg/internal/wordpress/pod_template.go [hunk 1]
```diff
@@ -347,6 +397,32 @@ func (wp *Wordpress) mediaContainers() []corev1.Container {
...
+	installCredentials := wp.installCredentials()
+
+	if installCredentials != nil {
+		scheme := "http"
+		if len(wp.Spec.TLSSecretRef) > 0 {
+			scheme = "https"
+		}
+		url := fmt.Sprintf("%s://%s/", scheme, wp.Spec.Domains[0])
+
+		installCmd := []string{
```
Comment (calind) [inline pkg/internal/wordpress/pod_template.go]: I think you need a small bash script here to check that `wp core is-installed` first. I recall that `wp core install` fails if the site is already installed, leasing to pods not being started anymore.

Comment (vtemian) [reply pkg/internal/wordpress/pod_template.go]: No, it exists with `0` and with the message that the wordpress is already installed.

Comment (vtemian) [reply pkg/internal/wordpress/pod_template.go]: I thought to have something like ``` wp core install || true``` but is not necessary.

File: pkg/internal/wordpress/pod_template.go [hunk 2]
```diff
@@ -89,6 +98,81 @@ func (wp *Wordpress) hasExternalMedia() bool {
 		(wp.Spec.MediaVolumeSpec.S3VolumeSource != nil || wp.Spec.MediaVolumeSpec.GCSVolumeSource != nil)
 }
 
+func (wp *Wordpress) installCredentials() *wpBootstrapConfig {
```
Comment (calind) [inline pkg/internal/wordpress/pod_template.go]: Use the `.Spec.Bootstrap.Env` and `.Spec.Bootstrap.EnvFrom` as is and inject them into the init container.

**Comment by vtemian** [comment]
@calind can you check again?

**Pull request merged**